biome.json

```diff
@@ -0,0 +1,94 @@
+{
+    "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+    "files": {
+        "includes": ["src/**", "test/**"]
+    },
+    "assist": {
+        "actions": {
+            "source": {
+                "organizeImports": "on",
+                "useSortedAttributes": "on"
+            }
+        }
+    },
+    "css": {
+        "formatter": {
+            "enabled": true,
+            "indentWidth": 2
+        },
+        "linter": {
+            "enabled": true
+        },
+        "parser": {
+            "allowWrongLineComments": true
+        }
+    },
+    "formatter": {
+        "useEditorconfig": true,
+        "enabled": true,
+        "lineEnding": "lf",
+        "lineWidth": 100,
+        "indentStyle": "space",
+        "indentWidth": 4,
```

Comment:
Please, be more close to the original settings. Some of the changes are meaningful, some are meaningless. With the new `indentWidth`, we update most of the source lines.